src/options/is-nested.options.ts

```diff
@@ -16,4 +16,5 @@ export function isClass<T>(target: unknown): target is Constructor<T> {
 
 export interface IsNestedOptions extends BaseOptions {
     type: Constructor<unknown>,
+    hasXPropertiesDefined?: number,
```

Comment:
Maybe a better name here. Contact me for context on this option.

Reply:
Per comments in https://github.com/hippo-oss/class-decorators/pull/7

 - It's not clear what use cases this feature supports
 - It's not clear why this should be a library feature vs a customer validator in a specific application

Reply:
👍 Will close, more detail at: https://github.com/hippo-oss/class-decorators/pull/7#issuecomment-1483228313